tests/conftest.py

```diff
@@ -62,6 +71,14 @@ def event_loop():
     loop.close()
 
 
+@pytest.fixture(scope='module')
+def monkeymodule():
+    """Make monkeypatching available in a module scope."""
+    mpatch = pytest.MonkeyPatch()
+    yield mpatch
+    mpatch.undo()
```

Comment:
Apparently this can now (Pytest ≤ 6.2) be a bit simpler https://github.com/pytest-dev/pytest/issues/363#issuecomment-1335631998

```suggestion
@pytest.fixture(scope='module')
def monkeymodule():
    """Make monkeypatching available in a module scope."""
    with pytest.MonkeyPatch.context() as mp:
        yield mp
```

Reply:
Looks like we're on Pytest 8 now so this should be no problem.